src/main/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/LoansServiceImpl.java

```diff
@@ -59,7 +60,14 @@
         String uri = LOANS_URI.expand(transactionId, companyAccountsId).toString();
 
         try {
-            LoanApi[] loans = apiClient.smallFull().loansToDirectors().loans().getAll(uri).execute().getData();
+            LoanApi[] loans = Arrays.stream(apiClient.smallFull().loansToDirectors().loans().getAll(uri).execute().getData())
+                    .map(loan -> {
+                        if(StringUtils.isBlank(loan.getDirectorName())) {
+                            loan.setDirectorName("Not provided");
```

Comment:
Did we come to an agreement on whether "Not provided" would be used? As I remember chatting about it in a slack thread with Liam and others on Friday :)

Reply:
Linked by Faisal to a previous discussion with James M, all seems okay with use of Not provided. Happy to OK pr 👍